### DISCUSSION TITLE
HM-CC-RT-DN nur über die CCU steuern?

### DISCUSSION THREAD

**Discussion by calllib** [OpenCCU/OpenCCU#1632] [🇩🇪 Fragen&Antworten] [answered]
Kann man den Heizkörperthermostaten HM-CC-RT-DN auch ausschließlich über die Zentrale steuern? Ich habe am Gerät die Wahl zwischen manuellem und automatischem Modus. In der Zentrale werden mir bei den Geräteparametern auch die "Temperaturprofile" angezeigt, ich kann sie aber nur bis auf ein letztes löschen, das mir dann immer dazwischenfunkt, wenn ich doch eigentlich die Solltemperatur abhängig von Anwesenheiten etc. ausschließlich über die Zentrale steuern möchte, ähnlich wie beim HM-CC-TC.
Geht das mit diesem Thermostaten gar nicht?

**Comment by Baxxy13** [comment, marked as answer]
Wenn du das Thermostat im MANU-Modus lässt sollte dir kein Wochenprogramm dazwischenfunken.

**Reply by calllib** [reply]
Funktioniert, danke!
Warum das nun nicht in den Anleitungen steht? eq3 ist schon seltsam...
